clang/test/Preprocessor/sycl-macro-target-specific.cpp

```diff
@@ -8,9 +8,26 @@
 // RUN: | FileCheck --check-prefix=CHECK-NVPTX-NEG %s
 // RUN: %clang_cc1 %s -fsycl-is-device -triple spir64_fpga-unknown-unknown -E -dM \
 // RUN: | FileCheck --check-prefix=CHECK-NVPTX-NEG %s
+// RUN: %clang_cc1 %s -fsycl-is-device -triple amdgcn-amdhsa-amdhsa -E -dM \
+// RUN: | FileCheck --check-prefix=CHECK-NVPTX-NEG %s
 // CHECK-NVPTX: #define __NVPTX__
 // CHECK-NVPTX-NEG-NOT: #define __NVPTX__
```

Comment:
Here too, though I understand this is not a new test.

Reply:
Comment added